.rubocop.yml

```diff
@@ -16,22 +16,39 @@
 
 LineLength:
   Max: 140
+
 MethodLength:
   Max: 40
```

Comment:
Consider decrease the max to 30, since no code changes are required at this point.